### DISCUSSION TITLE
Enhancement: Stuff I'd like to suggest cause what else do I have to do

### DISCUSSION THREAD

**Discussion by BumbleFucquer** [FunkinCrew/Funkin#1660] [Ideas]
Improvements on the game
1. You mispelled evilsk8r's name in the boot-up of the game before it says "In Association With Newgrounds." It's spelled evilsker
2. Downscroll doesn't stay on when I come back to the game even when I leave it on in a previous session
3. Is it possible to limit the number of loading screens in between songs or have a quick black screen for songs in between the weeks but use the regular loading screen for going into songs? 
Ex:
Clicking Week 6-> Regular green background with parents staring menacingly->Senpai-> Quick Black loading screen-> 
Roses->Black loading screen->Thorns-> Week ends and pulls up the week selection screen
Doing this would just pace the cutscenes better after completing a song. Congrats on getting the Kickstarter at 2 mil!

Sincerely,
BumbleFucquer

P.S. I just realized I mispelled mispell lol

**Comment by NTS-YT** [comment]
Looks like some good suggestions tbh

**Comment by g8mz** [comment]
evilsk8r is called evilsker is fnf because the font they use at start up doesnt have numbers in yet. so for the meantime theyre replacing it with an e

also how did you even get week 7 downloaded

**Comment by steviegt6** [comment]
> evilsk8r is called evilsker is fnf because the font they use at start up doesnt have numbers in yet. so for the meantime theyre replacing it with an e
> 
> also how did you even get week 7 downloaded

Where did they ever claim they did such a thing?

**Comment by g8mz** [comment]
> > evilsk8r is called evilsker is fnf because the font they use at start up doesnt have numbers in yet. so for the meantime theyre replacing it with an e
> > also how did you even get week 7 downloaded
> 
> Where did they ever claim they did such a thing?

which do you mean? if its the week 7 downloaded i dont know how i thought that

if its the evilsk8r thing then its because its in the newgrounds excluvise.
